src/scss/constants.scss

```diff
@@ -269,6 +271,8 @@ $z-layers: (
   map-grid: 3,
   map-room--unclickable: 2,
   map-background: 1,
+  // Admin Space Card
+  space-card-link: $z-layer-space-card-link,
```

Comment:
Why is this even necessary? Are there cases of the cards being overlapped or overlapping some other element? Schedule? Chat?

Reply:
i've added that z-index since I had to put a negative margin to the card's body container, so it fits on different screen resolutions with a max length name in it, no other real reason to have it